neuralpredictors/layers/readouts/factorized.py

```diff
@@ -84,6 +113,8 @@ def normalized_spatial(self):
             weight = self.spatial
         if self.constrain_pos:
             weight.data.clamp_min_(0)
+        elif self.positive_spatial:
+            weight = torch.abs(weight)
```

Comment:
I think it would be more intuitive if `positive_spatial` would be implemented in the same way as `constrain_pos`. The overall result of the trained network parameters should be the same.

```suggestion
        if self.constrain_pos or self.positive_spatial:
            weight.data.clamp_min_(0)
```

Reply:
In tensorflow 1.x there was no clamp_min afaik, hence we used abs

Reply:
I actually would like to keep it as in the original code [here](https://github.com/aecker/cnn-sys-ident/blob/master/cnn_sys_ident/architectures/readouts.py#L37) 

Because clamping might influence the gradient descent (because its zeros) and this might influence the performance